src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -1162,20 +1177,15 @@ export default {
 		}
 
 		blockquote {
-			position: relative;
+			padding-left: 0;
+			padding-inline-start: 13px;
 			border-left: none;
-
-			&::before {
-				content: ' ';
-				position: absolute;
-				top: 0;
-				left: 0;
-				height: 100%;
-				width: 4px;
-				border-radius: 2px;
-				background-color: var(--color-border);
-			}
+			border-inline-start: 4px solid var(--color-border);
 		}
 	}
 }
+
+:deep(.rich-text--component) {
+	direction: ltr;
+}
```

Comment:
We still have a pattern all over the apps - avatar on the left side of the name (Conversations, Participants, e.t.c)
First thing - don't want to break it for one specific case
Second - mention-bubble has a lot of styles, which hardcoded with left-right - so should be fixed upstream when we get to that.

Reply:
> We still have a pattern all over the apps - avatar on the left side of the name (Conversations, Participants, e.t.c)

Alright, but FYI, It's not the same thing as others. Conversations and Participants are standalone elements and are more like a structure rather than a reading direction whereas, in a message, you are reading in a certain direction.

Reply:
Agree. But i still tend to proceed with that fix upstream. And then styles will be here => disable hardcoded ltr, what do you think?

Reply:
Yes, I agree. It's better to fix the upstream